src/main/scala/beam/router/model/BeamLeg.scala

```diff
@@ -30,7 +30,8 @@ case class BeamLeg(startTime: Int, mode: BeamMode, duration: Int, travelPath: Be
   }
 
   def scaleToNewDuration(newDuration: Int): BeamLeg = {
-    val newTravelPath = this.travelPath.scaleTravelTimes(newDuration.toDouble / this.duration.toDouble)
+    val ratio = if (newDuration == 0 || this.duration == 0) 0 else newDuration.toDouble / this.duration.toDouble
```

Comment:
Otherwise if we get `Double.NaN`
![image](https://user-images.githubusercontent.com/5107562/87250428-dff50b80-c48e-11ea-9155-117f458d52a4.png)